### DISCUSSION TITLE
[Question] Difference between the .dmg version and the AppStore version?

### DISCUSSION THREAD

**Discussion by herry23xet** [coteditor/CotEditor#1569] [Q&A] [answered]
### Description

Hello,
CotEditor is great!
However, I'm not clear if and what differences there are between the versions (.dmg directly from GitHub and MacAppStore)?

### To Reproduce

See Description.

### CotEditor version

4.7.1

### macOS version

14.2.1

**Comment by 1024jp** [comment, marked as answer]
There is no difference in the software itself except the updater. 

cf. https://coteditor.com/archives
